include/git2/common.h

```diff
@@ -157,6 +157,7 @@ typedef enum {
 	GIT_OPT_SET_SSL_CERT_LOCATIONS,
 	GIT_OPT_SET_USER_AGENT,
 	GIT_OPT_ENABLE_STRICT_OBJECT_CREATION,
+	GIT_OPT_ENABLE_SYMBOLIC_REF_TARGET_VALIDATION,
```

Comment:
Just to be consistent with the name above it, could this be `GIT_OPT_ENABLE_STRICT_SYMBOLIC_REF_CREATION`?